apps/spruce/src/pages/adminSettings/tabs/RestartTasksTab/RestartTasksButton.tsx

```diff
@@ -3,9 +3,6 @@ import { useMutation, useQuery } from "@apollo/client";
 import styled from "@emotion/styled";
 import Button, { Variant as ButtonVariant } from "@leafygreen-ui/button";
 import { ConfirmationModal } from "@leafygreen-ui/confirmation-modal";
-// @ts-expect-error - LG advises directly importing from this path for simple spinners to avoid importing Lottie.
-// The component works, but the export is incorrectly typed. See LG-5659.
-// https://github.com/mongodb/leafygreen-ui/blob/main/packages/loading-indicator/CHANGELOG.md#major-changes
 import { Spinner } from "@leafygreen-ui/loading-indicator/spinner";
```

Comment:
Tangentially related, can we update the `Spinner` import in spruce's `LoadingButton` as a quick win